apps/indexer-coordinator/src/project/project.service.ts

```diff
@@ -174,6 +174,9 @@ export class ProjectService {
       project = await this.addProject(id);
     }
 
+    baseConfig.usePrimaryNetworkEndpoint =
+      baseConfig.usePrimaryNetworkEndpoint === false ? false : true;
```

Comment:
Can this be rewritten in a more simple way?

Reply:
readability improved. please review.